ejabberd.te

```diff
@@ -31,12 +31,12 @@ auth_use_nsswitch(ejabberd_t)
 corecmd_exec_bin(ejabberd_t)
 corecmd_exec_shell(ejabberd_t)
 
-corenet_tcp_bind_epmd_port(ejabberd_t)
+epmd_query(rabbitmq_t)
```

Comment:
You have here rabbitmq_t domain, which mean that you allowing rules in this macro for rabbitmq_t instead of ejabberd_t, but I still don't understand why you create epmd_query() interface. Do you have some BZ for this?